src/components/ButtonsBox.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+class ButtonsBox extends React.PureComponent {
+  render() {
+    return (
+      <div className="buttons">
+        <button type="submit">AC</button>
+        <button type="submit">+/-</button>
+        <button type="submit">%</button>
+        <button type="submit" className="operator">÷</button>
+        <button type="submit">7</button>
+        <button type="submit">8</button>
+        <button type="submit">9</button>
+        <button type="submit" className="operator">x</button>
+        <button type="submit">4</button>
+        <button type="submit">5</button>
+        <button type="submit">6</button>
+        <button type="submit" className="operator">-</button>
+        <button type="submit">1</button>
+        <button type="submit">2</button>
+        <button type="submit">3</button>
+        <button type="submit" className="operator">+</button>
+        <button type="submit" className="span-two">0</button>
+        <button type="submit">.</button>
+        <button type="submit" className="operator">=</button>
+      </div>
+    );
```

Comment:
- [ ] Kindly make this and other components to be functional components just as the requirements say.

![comp](https://user-images.githubusercontent.com/53124548/219616309-d1213ac0-8266-43fa-98c0-ffef60ed6883.png)